pkg/exporter/consul_exporter.go

```diff
@@ -509,6 +509,10 @@ func (e *Exporter) collectOneHealthSummary(ch chan<- prometheus.Metric, serviceN
 				break
 			}
 		}
+		if entry.Service.Service != serviceName {
+			level.Warn(e.logger).Log("msg", "Skipping service instance because its registered to %s but belongs to %s service registration", entry.Service.Service, serviceName)
```

Comment:
This seems like it would log a lot of stuff on every scrape. We probably want this debug level.

```suggestion
			level.Debug(e.logger).Log("msg", "Skipping service instance because its registered to %s but belongs to %s service registration", entry.Service.Service, serviceName)
```

Reply:
nice suggestion thanks!